api-compact.yaml

```diff
@@ -120,8 +164,8 @@ page_size_qs_parameter: &page_size_qs_parameter
 
 # AWS API Gateway vendor extension point. This information is used
 #  by https://github.com/awslabs/aws-apigateway-importer.
-x-amazon-apigateway-integration: &apigateway_integration
-  responses: *apigateway_responses_with_mapping_template
+x-amazon-apigateway-integration-id-response: &apigateway_integration_id
```

Comment:
@saleem-latif Yep, but same issue as above, unfortunately: it may be confusing to see this then not be used for the new `enterprise_catalogs` endpoint, which requires a `uuid`.